source/core/ut_suite_manager.pkb

```diff
@@ -689,16 +747,13 @@ create or replace package body ut_suite_manager is
             l_path_item.object_name,
             l_path_item.procedure_name,
             a_suites,
-            a_random_seed
+            a_random_seed,
+            a_tags
           );
         if a_suites.count = l_suites_count then
-          if l_path_item.suite_path is not null then
-            raise_application_error(ut_utils.gc_suite_package_not_found,'No suite packages found for path '||l_schema||':'||l_path_item.suite_path|| '.');
-          elsif l_path_item.procedure_name is not null then
-            raise_application_error(ut_utils.gc_suite_package_not_found,'Suite test '||l_schema||'.'||l_path_item.object_name|| '.'||l_path_item.procedure_name||' does not exist');
-          elsif l_path_item.object_name is not null then
-            raise_application_error(ut_utils.gc_suite_package_not_found,'Suite package '||l_schema||'.'||l_path_item.object_name|| ' does not exist');
-          end if;
+          raise_application_error(ut_utils.gc_suite_package_not_found,build_no_test_error_msg(
```

Comment:
Before the change, this was not throwing exception if running on a whole schema (with `ut.run()`) and schema has no tests.
This will be a breaking change in behavior as empty schema, schema without tests will start throwing exceptions.
Seems like we are missing a test to document existing behavior.